napari/components/viewer_model.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, title='napari'):
 
         # Initial dimension must be set to at least the number of visible
         # dimensions of the viewer
-        self.dims = Dims(2)
+        self.dims = Dims(3)
```

Comment:
you should revert this and will instead need to increase the dims in the call to the volume layer. i will comment below